cms/static/js/factories/login.js

```diff
@@ -12,6 +12,15 @@ define(['jquery.cookie', 'utility'], function() {
             });
         }
 
+        // Clear the login error message when credentials are edited
+        $('input#email').on('input',function() {
+            $('#login_error').removeClass('is-shown');
```

Comment:
Thanks for looking this over, @singingwolfboy .  

Removing `is-shown` does work correctly in my testing (OSX with both Safari and Chrome).  I was following some similar code in [the lms login page] (https://github.com/edx/edx-platform/blob/f6186aeda48e04fa8fae3b4310f0ed31e608b535/lms/templates/login.html#L54).

I tried your first suggestion as an alternative: adding the `is-hidden` class to the error element does *not* make it disappear.  I have not investigated why.

Your second suggestion, calling `.hide()` on the error element, does make the element disappear, but then it does not reappear later upon further errors.  The appearance and re-appearance of the error is governed by [code in `login.js`](https://github.com/edx/edx-platform/blob/0be8ab3aabef1e62614a1a17bdeb8da5a5396275/cms/static/js/factories/login.js#L33) that sets the `is-shown` class; removing `is-shown` seems like the logical inverse.

So the code in this PR is correct as far as I can tell.  Let me know if I should try something else.